bin/audit-svg-colors.js

```diff
@@ -16,34 +16,56 @@ const { readFileSync } = require( 'fs' );
 const path = require( 'path' );
 const PALETTE = require( '@automattic/color-studio' );
 const chroma = require( 'chroma-js' );
-const _ = require( 'lodash' );
+
+/**
+ * Native Sort - replacement of ._sortBy Lodash function
```

Comment:
I assume you meant `_.sortBy` since that's the way we invoke a Lodash function when it's globally declared.

```suggestion
 * Native Sort - replacement of _.sortBy Lodash function
```